src/quacc/recipes/vasp/core.py

```diff
@@ -16,6 +21,122 @@
     from quacc.utils.files import Filenames, SourceDirectory
 
 
+@job
+def nscf_job(
+    atoms: Atoms,
+    prev_dir: SourceDirectory,
+    bandgap: float | None = None,
+    #    nbands_factor: float | None = None,
+    preset: str | None = "BulkSet",
+    kpoints_mode: Literal["uniform", "line"] = "uniform",
+    calculate_optics: bool = False,
+    **calc_kwargs,
+) -> VaspSchema:
+    """
+    Carry out a non-self-consistent field (NSCF) calculation.
+
+    Parameters
+    ----------
+    atoms
+        Atoms object.
+    prev_dir: that is generally the folder (str | path) of the static_job
+    preset
+        Preset to use from `quacc.calculators.vasp.presets`.
+    copy_files
+        Files to copy (and decompress) from source to the runtime directory.
+    kpoints_mode
+        Type of k-points mode. Options are "uniform" or "line". "boltztrap" may
+        be implemented in future.
+    calculate_optics
+        Whether to calculate optical properties.
+    bandgap
+        The band gap of the material.
+    nbands_factor
+        It increases NBANDS. However, to use it, the vasprun.xml(.gz) must
+        provided in prev_dir.
+    **calc_kwargs
+        Custom kwargs for the Vasp calculator. Set a value to
+        `None` to remove a pre-existing key entirely. For a list of available
+        keys, refer to [quacc.calculators.vasp.vasp.Vasp][].
+
+    Returns
+    -------
+    VaspSchema
+        Dictionary of results from [quacc.schemas.vasp.vasp_summarize_run][].
+        See the type-hint for the data structure.
+    """
+
+    calc_defaults = {
+        "lorbit": 11,
+        "lwave": False,
+        "lcharg": False,
+        "nsw": 0,
+        "isym": 0,
+        "icharg": 11,
+        "kspacing": None,
+        "nedos": 5001,
+    }
+
+    # check the expected files in prev_dir
+    basics_to_copy = ["CHGCAR*", "WAVECAR*", "vasprun.xml*"]
```

Comment:
We do not need (and should not) copy the `vasprun.xml` file to the new VASP working directory.